enterprise/migrations/0136_auto_20210629_2129.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 2.2.20 on 2021-06-29 21:29
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+import simple_history.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('enterprise', '0135_adminnotification_adminnotificationfilter_adminnotificationread'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='systemwideenterpriseuserroleassignment',
+            name='role',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='system_wide_role_assignments', to='enterprise.SystemWideEnterpriseRole'),
+        ),
+        migrations.CreateModel(
+            name='HistoricalSystemWideEnterpriseUserRoleAssignment',
+            fields=[
+                ('id', models.IntegerField(auto_created=True, blank=True, db_index=True, verbose_name='ID')),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created')),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified')),
+                ('applies_to_all_contexts', models.BooleanField(default=False, help_text='If true, indicates that the user is effectively assigned their role for any and all contexts. Defaults to False.')),
+                ('history_id', models.AutoField(primary_key=True, serialize=False)),
+                ('history_date', models.DateTimeField()),
+                ('history_change_reason', models.CharField(max_length=100, null=True)),
+                ('history_type', models.CharField(choices=[('+', 'Created'), ('~', 'Changed'), ('-', 'Deleted')], max_length=1)),
+                ('enterprise_customer', models.ForeignKey(blank=True, db_constraint=False, help_text='The EnterpriseCustomer for which the role is assigned for the user.  If creating a new assignment, you must first provide a user email and role, then click "Save and continue editing" BEFORE selecting from this dropdown.', null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='enterprise.EnterpriseCustomer')),
```

Comment:
If you wanted to make this line more readable, you could save the help_text as a variable earlier

Reply:
This is just what django generates. It's a good suggestion in general, but I think for a migration it's not super necessary. No one is looking at this for the help text lol.